app/stats/knights.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+from app.stats.armor import Armour
+from app.stats.weapon import Weapon
+from app.stats.potion import Potion
+
+
+class Knights:
+    knights = {}
+
+    def __init__(
+            self,
+            name: str,
+            power: int,
+            hp: int,
+            armour: Optional[list[Armour]] = None,
+            weapon: Optional[Weapon] = None,
+            potion: Optional[Potion] = None
+    ) -> None:
+        self.protection = 0
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion
+        Knights.knights[name] = self
+
+    @classmethod
+    def registration(cls, knights: dict) -> list:
+        return [cls(
+            name=stat["name"],
+            power=stat["power"],
+            hp=stat["hp"],
+            armour=Armour.armour_registration(stat["armour"]),
+            weapon=Weapon.weapon_registration(stat["weapon"]),
+            potion=Potion.potion_registration(stat["potion"]))
+            for knight, stat in knights.items()]
+
+    def apply_armour(self) -> int:
```

Comment:
No need to return 
The same in apply_weapon

Reply:
apply_potion and BATTLE part don`t work without return in apply_armour and apply_weapon. :(
How can I change it?